ubbdadm/main.c

```diff
@@ -96,6 +99,7 @@ static void usage(int status)
 		printf("\
 			ubbdadm --command map --type file --filepath PATH --devsize SIZE\n\
 			ubbdadm --command map --type rbd --pool POOL --image IMANGE \n\
+			ubbdadm --command map --type ssh --hostname HOST --filepath REMOTE_PATH --devsize SIZE --num-queues N\n\
```

Comment:
this not support ssh port ? default ssh port is 22, and user can assign other port like:5522.